### PR TITLE
Gimp fix after replacing ilmbase by imath

### PR DESCRIPTION
**Pull request by shuryanc**
Due to replacement of ilmbase by imath, gimp needs to be fixed.

### CODE REVIEW

File: components/image/gimp/pkg5 [hunk 1]
```diff
@@ -20,7 +21,7 @@
         "library/desktop/pango",
         "library/glib2",
         "library/graphics/libmng",
-        "library/ilmbase",
+        "library/imath",
         "library/lcms2",
```
Comment (AndWac) [inline components/image/gimp/pkg5:24]: Did you run "gmake REQUIRED_PACKAGES" to update the list?
I run it and it didn't detect the dependency on imath (although installed).

**Comment by shuryanc** [comment]
Basically, imath should be needed by the packages that required ilmbase. But seems GIMP doesn't need imath since openexr 3.1.5.

**Comment by AndWac** [comment]
I'd rather say gimp doesn't detect it anymore. I expect this to be an optional feature.

**Comment by shuryanc** [comment]
I think I understand the reason. gimp requires libIex.so, which was in ilmbase. In v3.1.5, the libIex.so is in openexr instead of imath.

**Comment by AndWac** [comment]
This is now obsolete as gimp has been updated.

**Comment by AndWac** [comment]
We can only proceed if you fix the conflicts here and for gst-plugins-bad1. These packages have been changed after your PR's have been created.

**Comment by shuryanc** [comment]
Sure. Will update this commit. Just busy with life recently and get things delayed.

**Comment by shuryanc** [comment]
This is fixed.

**Pull request merged**